src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -240,6 +240,27 @@ private BoundAttribute BindAttributeCore(AttributeSyntax node, NamedTypeSymbol a
                     {
                         Error(diagnostics, ErrorCode.ERR_AttributeCtorInParameter, node, attributeConstructor.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat));
                     }
+
+                    for (var i = 0; i < boundConstructorArguments.Length; i++)
+                    {
+                        var parameter = GetCorrespondingParameter(i, attributeConstructor.Parameters, argsToParamsOpt, expanded);
+                        if (parameter is null)
+                        {
+                            continue;
+                        }
+
+                        var parameterType = expanded && parameter.IsParams && parameter.Type is ArrayTypeSymbol arrayType && arrayType.IsSZArray ? arrayType.ElementType : parameter.Type;
```

Comment:
>expanded && parameter.IsParams && parameter.Type is ArrayTypeSymbol arrayType && arrayType.IsSZArray [](http://example.com/codeflow?start=44&length=100)

Consider adding parentheses around the condition, so the order of evaluation is clear.

Reply:
Added. Also realized I can use a pattern here which saves some duplicate references, so I did that.